### PR TITLE
Tb factory bot

### PR DESCRIPTION
**Pull request by TravisBorgsmiller**
Added factory bot configuration gems and set up 
Added factories including user model which hasn't had a migration for it yet

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -60,20 +60,26 @@ GEM
       coffee-script-source
       execjs
     coffee-script-source (1.12.2)
-    concurrent-ruby (1.1.5)
-    crass (1.0.5)
+    concurrent-ruby (1.1.6)
```
Comment (cgaddis36) [inline Gemfile.lock:63]: Multiple areas where there are updated gemfiles

**Review by cgaddis36 — COMMENTED** [review]
Like the implementation of test helpers.  I see that we have to make a new file for each model.  Curious about all of the updated gems although I'm sure it is no big deal

File: Gemfile.lock (hunk 1 above)
Comment (KellyIB) [reply Gemfile.lock:63]: Thanks for taking the time to set this up for us.  I appreciate it, and I'm sure everyone else does too!

**Review by KellyIB — APPROVED** [review]
Interesting.  I see little updates to other gems in the gemfile, and notice that you set up order, user, merchant, and item files.  Thanks for setting this up, it looks like it will save a lot of typing!

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'support/factory_bot.rb'
```
Comment (adumortier) [inline spec/rails_helper.rb:11]: perfect, it's gonna make our tests more interesting and robust. Very nice addition!

File: spec/factories/user.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :random_user, class: User do
```
Comment (adumortier) [inline spec/factories/user.rb:2]: Let's go over what this does together

**Review by adumortier — APPROVED** [review]

**Pull request merged**